lib/npg_warehouse/loader/pacbio/run.pm

```diff
@@ -265,7 +265,8 @@ sub _well_qc_info {
     $qc{'p2_num'}                      = $qc_all->{'Productivity 2'};
     $qc{'adapter_dimer_percent'}       = $qc_all->{'Adapter Dimers (0-10bp) %'};
     $qc{'short_insert_percent'}        = $qc_all->{'Short Inserts (11-100bp) %'};
-    $qc{'control_num_reads'}           = $qc_all->{'Number of of Control Reads'};
+    $qc{'control_num_reads'}           = defined $qc_all->{'Number of of Control Reads'} ?
+      $qc_all->{'Number of of Control Reads'} : $qc_all->{'Number of Control Reads'};
```

Comment:
I love this.

Reply:
@ces can we get a comment here for line 268 that highlights that it's a workaround for smrtlink? The commit message makes it plain, but I expect this code to be laying around for at least another decade